LineageABCmodel.jl

```diff
@@ -3602,7 +3612,7 @@ function ABCwritelineagestatetotext( lineagetree::Lineagetree, state_hist::Array
         write( myfile, @sprintf("comment:     \t%s\n", uppars.comment) )
         write( myfile, @sprintf("chaincomment:\t%s\n", uppars.chaincomment) )
         write( myfile, @sprintf("timestamp:   \t%04d-%02d-%02d_%02d-%02d-%02d\n", year(uppars.timestamp),month(uppars.timestamp),day(uppars.timestamp), hour(uppars.timestamp),minute(uppars.timestamp),second(uppars.timestamp)) )
-        write( myfile, @sprintf("model:       \t%d\n", uppars.model) )
+        write( myfile, @sprintf("model:       \t%s\n", uppars.model) )

```

Comment:
I couldn't help but notice that this same writing/parsing business is repeated multiple times (I counted at least 4 of them) across the codebase, which is a **_strong_** indication functions should have been used to reduce [duplication](https://en.wikipedia.org/wiki/Duplicate_code) (duplication is error-prone and time-consuming, having to find, analyse and fix identical code snippets in multiple places in a consistent way).  I didn't undertake such an effort in order to minimise the changes: refactoring the code without any mean to make sure functionalities are preserved is basically a roulette.